supplementary_style_guide/glossary_terms_conventions/general_conventions/m.adoc

```diff
@@ -431,6 +441,17 @@ Azure CLI 2.0 is the most current command-line interface and is replacing Xplat-
 
 *See also*: xref:windows-server[Windows Server]
 
+[[migrate]]
+==== image:images/caution.png[with caution] migrate (verb)
+*Description*:  To _migrate_ data is to move data automatically from one location to another, usually to proactively or reactively improve cluster performance. OpenShift Container Platform uses the phrases _auto-migrate_ or _automatically migrate_ in this use case.
+In virtualization, _live migration_ refers specifically to a virtual machine being moved from one host to another while remaining operational.
+
+*Use it*: with caution - for clarity, use the full term first before shortening to _migrate_.
```

Comment:
Add the part about using the full term first before shortening to the definition instead.

Reply:
I'm going to yank this entry. It seems like the other terms might be actually what should be brought in as entries. CC @lpettyjo @laubai if you feel strongly that these terms should be added, let us know if we should add them in a followup PR